.azure-pipelines/warden.yml

```diff
@@ -24,9 +24,33 @@ jobs:
       python setup.py sdist -d $(Build.ArtifactStagingDirectory) --format zip
     displayName: 'Build Package'
 
-  - script: |
-      twine check $(Build.ArtifactStagingDirectory)/*
-    displayName: 'Verify Readme'
+  - task: CopyFiles@2
+    displayName: 'Copy files for github release'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)'
+      Contents: 'scripts/**/*.py'
+      TargetFolder: '$(Build.BinariesDirectory)/sdk-tools'
+      OverWrite: true
+      flattenFolders: true
```

Comment:
I'm not sure we should flatten and overwrite that seems to potentially cause issues. I would just keep the same structure and if we need to alter it we should.